DependencyInjection/Configuration.php

```diff
@@ -48,6 +48,16 @@ public function getConfigTreeBuilder()
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('filesystem')
```

Comment:
I'm not sure I like the way you fixed it with the other options btw, as it currently means that XML would look like:

````xml
<repository>
    <doctrine-phpcr-odm-repository basepath="..." />
</repository>
````